### PR TITLE
included minio in setup and teardown

### PR DESCRIPTION
**Pull request by murpheux**

### CODE REVIEW

File: openshift/setup-teardown/params/COMMON_SETTINGS/api/api.config [hunk 1]
```diff
@@ -1,4 +1,5 @@
 API_NODEJS_BUILD_NAME=eagle-api
+PI_MINIO_BUILD_NAME=eagle-api-minio
```
Comment (ActionAnalytics) [inline openshift/setup-teardown/params/COMMON_SETTINGS/api/api.config]: Minor copy/paste, looks like the first char got clipped out

File: openshift/setup-teardown/params/COMMON_SETTINGS/api/api.config [hunk 2]
```diff
@@ -7,6 +8,9 @@ API_BC_NODEJS_TEMPLATE_FILENAME=eagle-api.bc.json
 API_BC_NODEJS_PARAMS=api-1-nodejs-bc.params
 API_DC_MINIO_TEMPLATE_FOLDER_URL=https://raw.githubusercontent.com/your-github-org/your-api-repo/develop/openshift/templates/
 API_DC_MINIO_TEMPLATE_FILENAME=api-minio.dc.yaml
+API_BC_MINIO_TEMPLATE_FOLDER_URL=https://raw.githubusercontent.com/your-github-org/your-api-repo/develop/openshift/templates/
+API_BC_MINIO_TEMPLATE_FILENAME=minio-bc.json
```
Comment (ActionAnalytics) [inline openshift/setup-teardown/params/COMMON_SETTINGS/api/api.config:10]: let's put this before the dc just so it flows better for the reader

**Review by ActionAnalytics — CHANGES_REQUESTED** [review]
Looks good, minor updates and should be gtg

File: openshift/setup-teardown/params/COMMON_SETTINGS/api/api.config (hunk 1 above)
Comment (murpheux) [reply openshift/setup-teardown/params/COMMON_SETTINGS/api/api.config]: Done

File: openshift/setup-teardown/params/COMMON_SETTINGS/api/api.config (hunk 2 above)
Comment (murpheux) [reply openshift/setup-teardown/params/COMMON_SETTINGS/api/api.config:10]: Done

**Review by ActionAnalytics — APPROVED** [review]

**Pull request merged**